proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtils.java

```diff
@@ -307,12 +361,16 @@ private static String fromAnyValue(AnyValue anyValue) {
       return values.stream().map(OtlpProtobufUtils::fromAnyValue)
           .collect(Collectors.joining(", ", "[", "]"));
     } else if (anyValue.hasKvlistValue()) {
-      if (OTLP_DATA_LOGGER.isLoggable(Level.FINEST)) {
-        OTLP_DATA_LOGGER.severe("Encountered KvlistValue but cannot convert to String");
-      }
+      log(Level.SEVERE, "Encountered KvlistValue but cannot convert to String");
     } else if (anyValue.hasBytesValue()) {
       return Base64.getEncoder().encodeToString(anyValue.getBytesValue().toByteArray());
     }
     return "<Unknown OpenTelemetry attribute value type " + anyValue.getValueCase() + ">";
   }
+
+  private static void log(Level level, String message) {
+    if (OTLP_DATA_LOGGER.isLoggable(Level.FINEST)) {
```

Comment:
`Level.FINEST` should be `level` ?

Reply:
Good question, of which I'm not sure about the answer. We were following the same pattern established for Jaeger and Zipkin that allow customers to enable special loggers: https://docs.wavefront.com/tracing_integrations.html#enable-logs.

Actual usage of those loggers in the [Jaeger](https://github.com/wavefrontHQ/wavefront-proxy/blob/b89f51f1d43c71f4da1b2e9bbabadf68b70f59b5/proxy/src/main/java/com/wavefront/agent/listeners/tracing/JaegerProtobufUtils.java#L253-L257) and [Zipkin](https://github.com/wavefrontHQ/wavefront-proxy/blob/master/proxy/src/main/java/com/wavefront/agent/listeners/tracing/ZipkinPortUnificationHandler.java#L375-L378) listeners show that they check for `FINEST` level but log at `INFO`.

Reply:
Yeah, I need to review all the proxy login.

Reply:
I switched to using `OTLP_DATA_LOGGER.finest()` everywhere, as we'll be instructing users to set the logger's level to `ALL` when enabling this output.